src/amber/cli/templates/model/granite/src/models/{{name}}.cr.ecr

```diff
@@ -1,6 +1,7 @@
 require "granite_orm/adapter/<%= @database %>"
 
 class <%= @name.capitalize %> < Granite::ORM::Base
+  include Amber::Models::Granite
```

Comment:
yuck, don't like this at all.  Anyway of avoiding special Granite logic in Amber?

Reply:
We can have the initializer and set_attributes defined in Granite. I thought of that, but this is specific to Amber.

This will clean everything

Reply:
ok,  I'm torn.  This feels wrong to me but I see your argument for putting it here.  Are you ok with moving it to granite?